app/scenes/Login/Provider.js

```diff
@@ -104,7 +106,7 @@ class Provider extends React.Component<Props, State> {
           icon={<AuthLogo providerName={id} />}
           fullwidth
         >
-          Continue with {name}
+          {t("Continue with {{ name }}", { name: name })}
```

Comment:
```suggestion
          {t("Continue with {{ authProviderName }}", { authProviderName: name })}
```

A lesson I'm learning – being more specific here really helps the translators :)